Dockerfile_bundle

```diff
@@ -0,0 +1,16 @@
+FROM node:8.1.2-alpine
+
+ENV HOME=/usr/windbreaker
+
+RUN mkdir -p $HOME
+
+WORKDIR $HOME
+
+# install git
+RUN apk update && apk upgrade && \
+  apk add --no-cache git openssh
+
+ADD . $HOME
```

Comment:
Can we specify the volumes in windbreaker-deploy's docker-compose.yml file and use `ADD package.json $HOME` instead? If so, we can move the entrypoint/command to the docker-compose files and this file becomes identical to the other dockerfile. (same for other repos)

Reply:
Fixed.